.travis.yml

```diff
@@ -4,16 +4,17 @@ language: node_js
 node_js:
   - "lts/*"
 cache:
-  directories:
-    - "node_modules"
```

Comment:
When I run `yarn` locally, it repopulates `node_modules`. I'm wondering if we should keep caching it to speed up builds.

Reply:
Yes, it will still be cached.

```
cache:
  directories:
    - "node_modules"
  yarn: true
```

and 

```
cache:
  yarn: true
```

are equivalent.